packages/ses/src/error/note-log-args.js

```diff
@@ -89,26 +93,31 @@ const spliceOut = cell => {
  * rather than a `WeakMapStore`-like interface. To work before `lockdown`,
  * the implementation must use `freeze` manually, but still exhausively.
  *
- * It does not hold onto anything weakly. The only sense in which it is
- * WeakMap-like is that it does not support enumeration.
+ * It implements the WeakMap interface, and holds its keys only weakly.
+ * However, the spine of the doubly-linked ring (up to `budget` cells in length)
+ * is held strongly.
  *
- * TODO: Make parameterized `Key` and `Value` template types
+ * It may pin more than `budget` entries in memory if the user of the
+ * LRUCacheMap holds a key strongly, or the garbage collector has not yet
+ * emptied the unreferenced keys from the internal weakmaps.
```

Comment:
I'd be more precise here and say that some internal data may be left associated with a key even after it falls off the cache, but the values will not be held unless both the key is held by the user and the key hasn't fallen off the cache.